### PR TITLE
Fix some PHP compilecheck bugs

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/php/discovery_fragment.snip [hunk 1]
```diff
@@ -248,8 +248,8 @@
   optParams
 @end
 
-@private requestBodyName()
-  requestBody
+@private postBodyName()
+  postBody
```
Comment (tcoffee-google) [inline src/main/resources/com/google/api/codegen/php/discovery_fragment.snip]: Are there actual errors caused by this variable name? I looked everywhere for documentation of such and found nothing.

Comment (saicheems) [inline src/main/resources/com/google/api/codegen/php/discovery_fragment.snip]: Yeah... I think the compilecheck script is doing 1:1 comparisons of the parameter names

Comment (tcoffee-google) [inline src/main/resources/com/google/api/codegen/php/discovery_fragment.snip]: Per offline discussion, let's keep our original name and rewrite it in the compile checker like the one for Python.

Comment (saicheems) [inline src/main/resources/com/google/api/codegen/php/discovery_fragment.snip]: Done

**Comment by vchudnov-g** [comment]
If/when the compilecheck passes, LGTM

**Comment by saicheems** [comment]
(compile check does not succeed because the client libraries are lagging the discovery doc)

**Comment by vchudnov-g** [comment]
Is that the only issue we seem to be having? In that case, go ahead and submit. We'll get the PHP libraries updated.

**Comment by vchudnov-g** [comment]
(BTW, if you have a list handy of the APIs that fail due to the libraries being out of date, please send it my way.)

**Pull request merged**